woff1/tools/validator/validator.py

```diff
@@ -950,7 +949,7 @@ def _testTableDirectoryPadding(data, reporter):
         if paddingLength:
             paddingOffset = offset + length
             padding = data[paddingOffset:paddingOffset+paddingLength]
-            expectedPadding = "\0" * paddingLength
+            expectedPadding = ("\0" * paddingLength).encode()
```

Comment:
String data needs to be encoded in order to be able to perform binary data comparison in Python 3.